cmd/svcat/plan/get_cmd.go

```diff
@@ -90,23 +100,46 @@ func (c *getCmd) getAll() error {
 		return fmt.Errorf("unable to list classes (%s)", err)
 	}
 
+	if c.classFilter != "" {
+		if !c.lookupByUUID {
+			// Map the external class name to the class name.
+			for _, class := range classes {
+				if c.classFilter == class.Spec.ExternalName {
+					c.classFilter = class.Name
```

Comment:
Another svcat design is that evaluation of flags, and figuring out what exactly the user supplied value is (a uuid or an external name) is handled in the Validate function. See how plan uuid and name are handled for an example.

https://github.com/kubernetes-incubator/service-catalog/blob/8f986ae4a2bb7f6afe083b6ab3740635e2f571d9/cmd/svcat/plan/get_cmd.go#L61-L71

So instead of deciding if classFilter is a uuid or name in this function, it should be performed in `Validate` and the result saved into disambiguated fields on the command, for example:

```golang
type getCmd struct {
	*command.Context
	lookupByUUID bool
	uuid         string
	classFilter  string
	name         string
 
        classUUID string
        className string
}

```

Reply:
Done.